src/data/list/nodup.lean

```diff
@@ -330,6 +330,20 @@ lemma nodup.pairwise_of_set_pairwise {l : list α} {r : α → α → Prop}
   (hl : l.nodup) (h : {x | x ∈ l}.pairwise r) : l.pairwise r :=
 hl.pairwise_of_forall_ne h
 
+lemma nodup_iff {α : Type} {l : list α} :
```

Comment:
```suggestion
lemma nodup_iff_nth_ne_nth {α : Type} {l : list α} :
```
There are many things that `nodup_iff` could mean, so you should be more specific.

Can you move this just below `nodup.nth_le_inj_iff`, so that it is close to similar results?

Reply:
done